earthquakes.py

```diff
@@ -3,7 +3,10 @@
 # However, we will use a more powerful and simpler library called requests.
 # This is external library that you may need to install first.
 import requests
+import os # The 'os' module gives us all the tools we need to search in the file system
 
+os.getcwd() # Use the 'getcwd' function from the 'os' module to find where we are on disk.
```

Comment:
I'm not sure why you need `os` here - to see where the plot is being saved perhaps?